azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/telemetry/AppInsightsProxy.java

```diff
@@ -24,15 +34,42 @@
     protected boolean isEnabled = true;
 
     public AppInsightsProxy(final TelemetryConfiguration config) {
-        // InstrumentationKey will be read from ApplicationInsights.xml
-        client = new TelemetryClient();
+        client = new TelemetryClient(readConfigurationFromFile());
```

Comment:
If there is no file named applicationinsights.properties, then it will fallback to applicationinsights.xml?
new TelemetryClient(readConfigurationFromFile()); will follow above convention?